app/Repositories/Cts/ValidationPositionRepository.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Repositories\Cts;
+
+use App\Models\Cts\Member;
+use App\Models\Cts\ValidationPosition;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+class ValidationPositionRepository
+{
+    public function findByPositionId(int $id)
+    {
+        return ValidationPosition::find($id)->first();
```

Comment:
Don't need the first call here - find will return the first or null. See my other comment too.

Reply:
8bdefee8